portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase

```diff
@@ -198,6 +263,521 @@ definition {
+
+		task ("Filter blogs entry by category") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
+
+			AssetListsAdmin.addCategoriesFilter(
+				categoryNameList = "Audience 1,Stage 1",
+				queryAndOperator = "All",
+				queryContains = "Contains");
+		}
+
+		task ("Add a Collection Display to page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Page Name",
+				type = "content");
+
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addWidget(portletName = "Asset Publisher");
+
+			Button.clickPublish();
+		}
+
+		task ("Assert the correct asset is shown") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/test-page-name");
+
+			AssertTextEquals.assertPartialText(
+				key_assetTitle = "Blogs Entry Title",
+				locator1 = "AP#ASSET_ABSTRACT_TITLE",
+				value1 = "Blogs Entry Title");
+
+			AssertTextEquals.assertPartialText(
+				key_assetContent = "Blogs Entry Content",
+				locator1 = "APAsset#ASSET_CONTENT",
+				value1 = "Blogs Entry Content");
+		}
+	}
+
+	@description = "LPS-148882: Given a Blog Entry with Audience and/or Stage categories
+	And those categories are filtered at Dynamic Collection
+	When user clicks at View Items
+	Then the blog asset should be listed"
+	@priority = "3"
+	test CheckBlogFilterByCategoryAtCollection {
+		property portal.acceptance = "true";
+
+		task ("Add a global category") {
+			var categoryId1 = JSONCategory.addCategory(
+				groupName = "Global",
+				title = "Audience 1",
+				vocabularyName = "Audience");
+			var categoryId2 = JSONCategory.addCategory(
+				groupName = "Global",
+				title = "Stage 1",
+				vocabularyName = "Stage");
+		}
+
+		task ("Create a blog") {
+			HeadlessBlog.addBlogEntry(
+				assetCategoryIds = "${categoryId1},${categoryId2}",
+				content = "Blogs Entry Content",
+				groupName = "Test Site Name",
+				title = "Blogs Entry Title");
+		}
+
+		task ("Create a Dynamic Collection") {
+			JSONAssetlist.addDynamicAssetListEntryWithItemType(
+				groupName = "Test Site Name",
+				itemSubtype = "All Subtypes",
+				itemType = "Blogs Entry",
+				title = "Dynamic Collection Test");
+		}
+
+		task ("Create a first variation for the collection and filter web contents by category") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
+
+			AssetListsAdmin.addCategoriesFilter(
+				categoryNameList = "Audience 1,Stage 1",
+				queryAndOperator = "All",
+				queryContains = "Contains");
+		}
+
+		task ("Go to Content Dashboard and check the asset is being counted at graph") {
+			AssetListsAdmin.gotoEllipsisMenuItemInItemSelector(
+				fieldLabel = "Anyone",
+				menuItem = "View Items");
+
+			SelectFrame(locator1 = "IFrame#MODAL_BODY");
```

Comment:
You can add `SelectFrame` to the previous function. As it's opens a modal, anything poshi will do require the modal to be selected.